README.md

```diff
@@ -109,6 +109,28 @@ Check out all related information [here](GSSoC.md)
     1. [Python](https://realpython.com/python-virtual-environments-a-primer/#why-the-need-for-virtual-environments)
     2. [Conda](https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html)
 ------
+
+#### How to set me up on Slack?
+
+- Clone the repository
+- Create the **mapbot** database in mySQL
+  -  `mysql -uroot -p -hlocalhost`
+  - Enter root password when prompted
+  - `create database mapbot;`
+  - Verify creation of the database `show databases;`
+- Unzip the StanfordCoreNLP package in the repository and keep the file paths `stanford-corenlp-x.x.x.jar` and `stanford-corenlp-x.x.x-models.jar` handy.
+- Run `git update-index --assume-unchanged ENV/.env`
+- Fill the existing template in `ENV/.env` with the corresponding values following the `KEY=VALUE` format
+- Next create a **classic** slack app from [here](https://api.slack.com/apps?new_classic_app=1).
+- After creating a classic slack app and filling out some basic details, go to **Basic Information** section from the sidebar.
+- In the Basic Information page, copy the **Client ID** and **Client Secret** and paste these in the `ENV/.env` file as: `slack_client_id=Your Client ID` and `slack_client_secret=Your Client Secret`.
```

Comment:
Change this to 'Copy the **Client ID** and **Client Secret** and paste these in the `ENV/.env` file as: `SLACK_CLIENT_ID=Your Client ID` and `SLACK_CLIENT_SECRET=Your Client Secret`.'
Use all capitals for constants in `.env`